src/cmdline.c

```diff
@@ -96,7 +96,7 @@ int cmdline_run(const char **argv, int argc)
 	 * a null pointer in the array. Because of this, AppKit in OS X 10.10 will
 	 * dereference it, causing a segmentation fault.
 	 */
-	char** mutableArgv = malloc(argvsize);
+	const char** mutableArgv = malloc(argvsize);
```

Comment:
Fair point. GCC/clang complains about losing `const` when being passed to `argparse_size`, let me fix that.